File: Examples/CSharp/WorkingWithTaskLinks/GetCrossProjectTaskLinks.cs
namespace Aspose.Tasks.Examples.CSharp.WorkingWithTaskLinks
{
    using System;

    internal class GetCrossProjectTaskLinks
    {
        public static void Run()
        {
            var dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod());
            
            //ExStart:GetCrossProjectTaskLinks
            //ExFor: Project.TaskLinks
            //ExFor: TaskLink.CrossProjectName
            //ExSummary: Shows how to find cross project task links. 
            var project = new Project(dataDir + "GetCrossProjectTaskLinks.mpp");

            // Check cross project task links
            foreach (var taskLink in project.TaskLinks)
            {
                if (taskLink.IsCrossProject)
                {
                    Console.WriteLine(taskLink.CrossProjectName);
                }
            }
            //ExEnd:GetCrossProjectTaskLinks
        }
    }
}

File: Examples/VisualBasic/WorkingWithTaskLinks/GetCrossProjectTaskLinks.vb

Namespace WorkingWithTaskLinks
    Public Class GetCrossProjectTaskLinks
        Public Shared Sub Run()
            ' ExStart:GetCrossProjectTaskLinks
            ' Create project instance
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType.FullName)
            Dim project1 As New Project(dataDir + "GetCrossProjectTaskLinks.mpp")

            ' Check cross project task links
            For Each tsklnk As TaskLink In project1.TaskLinks
                If tsklnk.IsCrossProject Then
                    Console.WriteLine(tsklnk.CrossProjectName)
                End If
            Next
            ' ExEnd:GetCrossProjectTaskLinks
        End Sub
    End Class
End Namespace
